internal/cache/file/cache_handle.go

```diff
@@ -84,10 +83,13 @@ func (fch *CacheHandle) shouldReadFromCache(jobStatus *downloader.JobStatus, req
 		jobStatus.Name == downloader.Failed {
 		err := fmt.Errorf("%s: jobStatus: %s jobError: %w", util.InvalidFileDownloadJobErrMsg, jobStatus.Name, jobStatus.Err)
 		return err
-	} else if jobStatus.Offset < requiredOffset {
+	}
+
+	if jobStatus.Offset < requiredOffset {
```

Comment:
any advantage of moving this to `if` from `else if` ?

Reply:
having if with return is more readable than if else ladder, else is redundant here, as there is an if inside